docs/resources/integration_aws_account.md

```diff
@@ -0,0 +1,221 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_integration_aws_account Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage Datadog - Amazon Web Services integration.
+---
+
+# datadog_integration_aws_account (Resource)
+
+Provides a Datadog - Amazon Web Services integration resource. This can be used to create and manage Datadog - Amazon Web Services integration.
+
+## Example Usage
+
+```terraform
+# Create new integration_aws_account resource
+
+resource "datadog_integration_aws_account" "foo" {
+  account_tags   = ["env:prod"]
+  aws_account_id = "123456789012"
+  aws_partition  = "aws"
+  aws_regions {
+    include_all = true
+  }
+
+  auth_config {
+    aws_auth_config_role {
+      role_name = "DatadogIntegrationRole"
+    }
+  }
+  logs_config {
+    lambda_forwarder {
+      lambdas = ["arn:aws:lambda:us-east-1:123456789012:function:my-lambda"]
+      sources = ["s3"]
+    }
+  }
+  metrics_config {
+    automute_enabled          = true
+    collect_cloudwatch_alarms = true
+    collect_custom_metrics    = true
+    enabled                   = true
+    namespace_filters {
+      exclude_only = ["AWS/SQS", "AWS/ElasticMapReduce"]
+    }
+    tag_filters {
+      namespace = "AWS/EC2"
+      tags      = ["datadog:true"]
+    }
+  }
+  resources_config {
+    cloud_security_posture_management_collection = true
+    extended_collection                          = true
+  }
+  traces_config {
+    xray_services {
+      include_all = true
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `aws_account_id` (String) Your AWS Account ID without dashes.
+- `aws_partition` (String) AWS Account partition
+
+### Optional
+
+- `account_tags` (List of String) Tags to apply to all metrics in the account
+- `auth_config` (Block, Optional) (see [below for nested schema](#nestedblock--auth_config))
```

Comment:
This isn't optional :( are we able to manually override/edit this so it can say required in our docs?

Reply:
Adding this to the generate-docs exclusion list so we can fix this